internal/grpctest/tlogger.go

```diff
@@ -73,13 +74,20 @@ type tLogger struct {
 }
 
 func init() {
-	TLogger = &tLogger{errors: map[*regexp.Regexp]int{}}
+	tLoggerAtomic.Store(&tLogger{errors: map[*regexp.Regexp]int{}})
 	vLevel := os.Getenv("GRPC_GO_LOG_VERBOSITY_LEVEL")
 	if vl, err := strconv.Atoi(vLevel); err == nil {
-		TLogger.v = vl
+		if logger, ok := tLoggerAtomic.Load().(*tLogger); ok {
```

Comment:
It's okay to use the infallible type assertion here since we're sure of the type.
```go
logger := tLoggerAtomic.Load().(*tLogger)
```

After reading further, we can use `getLogger` here.